components/tidb_query/src/codec/collation/utf8mb4.rs

```diff
@@ -367,53 +376,69 @@ impl Collator for CollatorUtf8Mb4GeneralCi {
         Ok(n * std::mem::size_of::<u16>())
     }
 
+    fn sort_compare(a: &[u8], b: &[u8]) -> Result<Ordering> {
+        let sa = str::from_utf8(a)?.trim_end_matches(TRIM_PADDING_SPACE);
+        let sb = str::from_utf8(b)?.trim_end_matches(TRIM_PADDING_SPACE);
+        Ok(sa
+            .chars()
+            .map(general_ci_convert)
+            .cmp(sb.chars().map(general_ci_convert)))
+    }
+
+    fn sort_hash<H: Hasher>(_state: &mut H, _bstr: &[u8]) -> Result<()> {
+        unimplemented!()
```

Comment:
why is it unimplemented?

Reply:
Because it's to be deleted. I found it hard to maintain the code base during the unstable requirement.